TrafficCapture/replayerPlugins/jsonMessageTransformers/Es2OsPost23/src/main/java/org/opensearch/migrations/transform/Es2OsPost23.java

```diff
@@ -0,0 +1,9 @@
+package org.opensearch.migrations.transform;
+
+import io.burt.jmespath.jcf.JcfRuntime;
+public class Es2OsPost23 extends JsonJMESPathTransformerProvider {
+    public JsonJMESPathTransformer createTransformer(Object jsonConfig) {
+        String script = "{ \"settings\": settings, \"mappings\": payload.mappings.*.properties | [merge(@)] }";
+        return new JsonJMESPathTransformer(new JcfRuntime(), script);
```

Comment:
OK - now I see why you had to add the dependency... I setup the c'tor to take the dependency because we could be using JSON nodes from jackson or something else - and those themself will require additional dependencies.

Anyway, please add an override constructor that defaults to the JcfRuntime & use that instead...
```
    public JsonJMESPathTransformer(String script) {
        this(new JcfRuntime(), script);
    }
```